lib/std/fs.zig

```diff
@@ -2263,6 +2263,52 @@ pub fn realpathAlloc(allocator: *Allocator, pathname: []const u8) ![]u8 {
     return allocator.dupe(u8, try os.realpath(pathname, &buf));
 }
 
+const CopyFileError = error{SystemResources} || os.CopyFileRangeError || os.SendFileError;
+
+/// Transfer all the data between two file descriptors in the most efficient way.
+/// No metadata is transferred over.
```

Comment:
Can you answer these 2 questions in the doc comments?
 * Are the seek positions required to be in any particular state before calling?
 * Are the seek positions affected by this function call?